src/barretenberg/pedersen.rs

```diff
@@ -24,21 +24,21 @@ impl Pedersen for Barretenberg {
     }
 }
 
-#[test]
-fn pedersen_hash_to_point() -> Result<(), Error> {
-    let barretenberg = Barretenberg::new();
-    let (x, y) = barretenberg.encrypt(vec![FieldElement::zero(), FieldElement::one()])?;
-    let expected_x = FieldElement::from_hex(
-        "0x11831f49876c313f2a9ec6d8d521c7ce0b6311c852117e340bfe27fd1ac096ef",
-    )
-    .unwrap();
-    let expected_y = FieldElement::from_hex(
-        "0x0ecf9d98be4597a88c46a7e0fa8836b57a7dcb41ee30f8d8787b11cc259c83fa",
-    )
-    .unwrap();
-
-    assert_eq!(expected_x.to_hex(), x.to_hex());
-    assert_eq!(expected_y.to_hex(), y.to_hex());
-    Ok(())
-}
+// #[test]
+// fn pedersen_hash_to_point() -> Result<(), Error> {
```

Comment:
Let's restore these tests.

Reply:
Would we not want to have these tests now in mocha?